js/main.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-alert */
+/* eslint-disable no-undef */
+
+// player factory for create player objects
```

Comment:
**Kindly remove commented code, you can always use git history to access previous code right :slightly_smiling_face:**

Reply:
done